### PR TITLE
[TEVA-3965] default page width and filter page layout improvements

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-3965
https://dfedigital.atlassian.net/browse/TEVA-3897

- page width changed to use 1020px GDS recommendation
- pages with filters improved to have better fluid layout at all screen widths
- did do some minor SASS file refactoring/removal of unused styles

### CODE REVIEW

**Comment by Kirill-Titov-Designer** [comment]
The proposed design look good to me.

File: app/frontend/src/styles/application.scss [hunk 1]
```diff
@@ -12,12 +12,15 @@ $govuk-new-link-styles: true;
 
 @import '~govuk-frontend/govuk/all';
 
+@import 'base/grid';
 @import 'base/mixins';
+@import 'base/typography';
 @import 'base/utilities';
 @import 'global/variables';
```
Comment (cesidio) [inline app/frontend/src/styles/application.scss:19]: I woudn't keep this file if it's empty

File: app/frontend/src/styles/application.scss [hunk 2]
```diff
@@ -58,15 +50,9 @@ dd {
...
-    border-bottom: 6px solid $govuk-border-colour;
-    padding-bottom: govuk-spacing(2);
-  }
-}
-
-.govuk-footer__list {
-  padding-bottom: 40px;
+ul li + br,
+ul > br {
+  display: none;
```
Comment (alexbowen) [inline app/frontend/src/styles/application.scss:55]: these styles are very good example of visual testing being useful, they are so generic, im not what they do, what might change if i remove them! i cant search codebase for a classname

File: app/frontend/src/styles/application.scss (hunk 1 above)
Comment (cesidio) [reply app/frontend/src/styles/application.scss:19]: @import 'global/variables';

**Review by cesidio — APPROVED** [review]

**Pull request merged**